docs.html

```diff
@@ -0,0 +1,377 @@
+                <button type="button" class="btn btn-outline-secondary">Secondary</button>
+                <button type="button" class="btn btn-outline-success">Success</button>
+                <button type="button" class="btn btn-outline-danger">Danger</button>
+                <button type="button" class="btn btn-outline-warning">Warning</button>
+                <button type="button" class="btn btn-outline-light">Light</button>
+                <button type="button" class="btn btn-outline-dark">Dark</button>
+            </div>
+            <div class="code-snippet__container">
+                <iframe
+                    src="https://carbon.now.sh/embed/QCqNr6Pfm8Im5PNPer53"
+                    class="code-snippet"
+                    style="min-height: 13rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <h1 class="heading-sm txt-light">Different Sized buttons</h1>
+            <h1 class="heading-sm txt-dark">Large , Medium , Small</h1>
+            <button type="button" class="btn btn-primary btn-lg">Large Button</button>
+            <button type="button" class="btn btn-primary btn-md">Medium Button</button>
+            <button type="button" class="btn btn-primary btn-sm">Small Button</button>
+            <div class="code-snippet__container">
+                <iframe
+                    src="https://carbon.now.sh/embed/jmBxTeeXvMwB54NWUoD3"
+                    class="code-snippet"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+        </div>
+        <div class="docs__container">
+            <div id="card">
+                <h1 class="heading-md txt-dark">Card</h1>
+                <h1 class="heading-sm txt-light">Standard Card , Horizantal card</h1>
+                <div class="component-preview__container-flex">
+                    <div class="card">
+                        <div class="card-img__container">
+                            <img src="/assets/image1.png" alt="sample" class="card-img">
+                        </div>
+                        <div class="card-details">
+                            <h2 class="card-heading">Heading</h2>
+                            <h2 class="card-sub-heading">Sub Heading</h2>
+                            <p class="card-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus, error accusantium eius fuga</p>
+                            <div class="card-price">
+                                <span class="price">Rs. 699</span>
+                                <span class="price-original">Rs. 999</span>
+                            </div>        
+                            <button type="button"class="btn btn-outline-primary">Buy Now</button>
+                        </div>
+                    </div>
+                    <div class="card card-hz">
```

Comment:
The landing page and docs are minimalist and look awesome!!
One thing I noticed is that the horizontal card has stretched to the full height of the parent container.

Reply:
Actually it was intentionally done, because if I give a height to the card then if there are contents which does not fit the card then it will overflow right? 
Any suggestion on how to change height dynamically then?

Reply:
You can give height as fit-to-content and maybe a specific width so that if there's a lot of content the card's height will change accordingly.

Reply:
That's why I have given only min-height:14rem to every card , and let the container height grow according the content

Reply:
Should I remove the minimum height?